op-program/zkWasm-emulator/wasi/wasi_exec_node.js

```diff
@@ -40,24 +36,9 @@ import path from 'node:path';
     }
   }
 
-  let max_mem = 0
-
-  const wasi_imports = wasi.getImportObject()
-  const previous_clock_time_get = wasi_imports.wasi_snapshot_preview1.clock_time_get
-  wasi_imports.wasi_snapshot_preview1.clock_time_get =  (clockId, precision, time) => {
-    let res = previous_clock_time_get(clockId, precision, time)
-    if (max_mem < instance.exports.memory.buffer.byteLength) {
-      max_mem = instance.exports.memory.buffer.byteLength
-    }
-    return res
-  }
-
-  instance = await WebAssembly.instantiate(wasm, {...wasi_imports,...hostio});
+  instance = await WebAssembly.instantiate(wasm, hostio);
   wasi.start(instance);
+})()
 
-  process.on("exit", ()=> {
```

Comment:
Is there a way to leave some report ( or make report-generated become an option)?

The rationale is the hostio for printing things have been removed, but we still need some observability for debug purpose, because note that we changed the compiler as well. We expect something just like memory watermark changes, pc changes or something similar: no code change in wasm itself but uses js env to provide some metrics.

Reply:
Currently, I've removed all the debug prints. I suggest using wasm_output to debug. Is it ok?